packages/devtools/src/SlotCSSRules.tsx

```diff
@@ -25,15 +26,22 @@ export const SlotCSSRules: React.FC<{ slot: string; atomicRules: AtomicRules[] }
   const rules = React.useMemo(() => getMonolithicCSSRules(atomicRules), [atomicRules]);
   const classes = useStyles();
 
+  const [expanded, setExpanded] = React.useState(true);
+  const toggleExpanded = () => setExpanded(v => !v);
+
   const { setHighlightedClass } = useViewContext();
   const undoHighlight = () => setHighlightedClass('');
 
   return (
     <>
-      <pre className={classes.slotName}>{slot}</pre>
-      <div className={classes.rules} onClick={undoHighlight}>
-        <MonolithicRulesView rules={rules} />
-      </div>
+      <pre className={classes.slotName} onClick={toggleExpanded}>
+        {slot}
+      </pre>
```

Comment:
I think that are missing there an indicator to indicate if it's opened:

![image](https://user-images.githubusercontent.com/14183168/166665833-b8c8da53-6a64-490d-b980-28083674e010.png)

Reply:
good idea, let me create a separate PR for this

Reply:
PR #116